python/freetensor/core/__init__.py

```diff
@@ -38,3 +38,8 @@
 from .optimize import optimize
 
 from .task_scheduler import TaskScheduler
+from .remote_task_scheduler import RemoteTaskScheduler, MultiMachineScheduler, MeasureTask
+
+from . import RPCTools
```

Comment:
I suggest `rpc_tools` instead of `RPCTools`, to be consistent with other modules.

Reply:
Or `rpc_utils`.